api-examples/hydra/prefix/prefixed-names.hydra

```diff
@@ -0,0 +1,15 @@
+PREFIX "api": <http://example.com/vocab#>
```

Comment:
This is the best I could get. Ideally I'd like to drop the quotes but `STRING` is the only terminal which doesn't collide with `FIELD_NAME` atm